charts/bctlquickstart/templates/quickstart/bctl-quickstart-job.yaml

```diff
@@ -5,10 +5,10 @@ metadata:
   name: {{ include "bctlquickstartchart.quickstartJobName" . }}
   labels:
     {{- include "bctlquickstartchart.labels" . | nindent 4 }}
-  {{- with .Values.annotations }}
   annotations:
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "1"
```

Comment:
Should we use 0 here?  From the docs:
> That behavior, though, is not considered binding and could change in the future.) It is considered good practice to add a hook weight, and set it to 0 if weight is not important
Ref: https://helm.sh/docs/topics/charts_hooks/

Reply:
a5b18ba